src/features/bivariate_color_manager/atoms/bivariateColorManagerData.ts

```diff
@@ -186,6 +201,54 @@ const layersFilterFunction: FilterFunction = (
   return Boolean(value.horizontal[indicator] || value.vertical[indicator]);
 };
 
+function compareArrays(array1: string[], array2: string[]): boolean {
```

Comment:
we also have function `arraysAreEqual` in `src/features/layers_in_area/atoms/areaLayersDetailsResource.ts` with the same body. It can be moved to `utils/common` i suppose